doc/UsersGuide/source/index.rst

```diff
@@ -14,7 +14,6 @@ Welcome to the UFS Weather Model User's Guide
    CodeOverview
    BuildingAndRunning
    InputsOutputs
-   SDFandNamelistExamplePractices
```

Comment:
This section is no longer there? I feel it could be very helpful.

Reply:
We moved this section to the CCPP Technical Documentation instead of here, see https://github.com/NCAR/ccpp-doc

Reply:
Thanks